packages/components/src/EditableItemList.test.tsx

```diff
@@ -18,9 +18,9 @@ function makeWrapper({
   validate = undefined,
 }: {
   items?: string[];
-  onAdd?: (string) => void;
-  onDelete?: (string) => void;
-  validate?: (string) => Error | null;
+  onAdd?: (item: string) => void;
+  onDelete?: (items: string[]) => void;
+  validate?: (item: string) => Error | null;
```

Comment:
Instead of redefining the types here, you could just do a `Partial<EditableItemListProps>`